[enterprise-4.9] Add 4.9 ShiftStack release notes (#36531)
openshift/openshift-docs: release_notes/ocp-4-9-release-notes.adoc

For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].

[id="ocp-4-9-networking-openshift-on-openstack-cloud-provider-options"]
==== Load balancer configuration as a cloud provider option for clusters on {rh-openstack}

> For clusters that run on {rh-openstack}, you can now configure Octavia for load balancing as a cloud provider option. 

Reviewer: Perhaps this is a bit confusing, as it was already possible to configure the load balancer previously. In 4.9, we've changed the default to enable the use of octavia when applicable, and have QE coverage to support multiple scenario.
There's also desire to support the Octavia load balancer in previous releases (provided manual configuration).

Author: ```suggestion
For clusters that run on {rh-openstack}, Octavia is now available as a default load balancer option. 
```

@mandre ?

// TODO: Link!

> // For more information, see xref:../

Reviewer 2: I added a known issue about this feature into the tracker PR recently. :/ Here's the BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2004542.

Author: TY. :\
